From 914553b69e7c2259239a8cab7e331f3cee8284b2 Mon Sep 17 00:00:00 2001
From: Stefan Hajnoczi <stefanha@redhat.com>
Date: Tue, 26 Aug 2014 17:01:40 -0500
Subject: [CHANGE 1/3] Fix TEXI section mark imbalance in qemu-img-cmd.hx
To: rhvirt-patches@redhat.com,
    jen@redhat.com

RH-Author: Stefan Hajnoczi <stefanha@redhat.com>
Message-id: <1409072501-28240-2-git-send-email-stefanha@redhat.com>
Patchwork-id: 60709
O-Subject: [PATCH v2 1/2] Fix TEXI section mark imbalance in qemu-img-cmd.hx
Bugzilla: 1116558
RH-Acked-by: John Snow <jsnow@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>

From: Jan Kiszka <jan.kiszka@siemens.com>

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
Signed-off-by: Blue Swirl <blauwirbel@gmail.com>
(cherry picked from commit 3c4c32101bdb4ee939e40a8e3d3f9d2a5d0ec053)
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
---
 qemu-img-cmds.hx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Jeff E. Nelson <jen@redhat.com>
---
 qemu-img-cmds.hx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qemu-img-cmds.hx b/qemu-img-cmds.hx
index 3a147d5..05af6ed 100644
--- a/qemu-img-cmds.hx
+++ b/qemu-img-cmds.hx
@@ -7,7 +7,7 @@ HXCOMM HXCOMM can be used for comments, discarded from both texi and C
 
 STEXI
 @table @option
-STEXI
+ETEXI
 
 DEF("check", img_check,
     "check [-f fmt] [--output=ofmt] [-r [leaks | all]] filename")
-- 
1.9.3

